rasa_nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -27,6 +22,10 @@
 
 if typing.TYPE_CHECKING:
     import tensorflow as tf
+    from rasa_nlu.config import RasaNLUModelConfig
```

Comment:
usually all that stuff gets imported where it was before. not sure which is the correct way to do it though :P

Reply:
I noticed in rasa_core files all rasa_core imports for types were imported in such if. I don't know what is the right way as well.

Reply:
interesting. @amn41 thoughts?

Reply:
not in all rasa_core files though

Reply:
this is required if you are using types from optional dependencies